pandas/core/arrays/interval.py

```diff
@@ -433,30 +434,33 @@ def from_arrays(cls, left, right, closed="right", copy=False, dtype=None):
             ),
         )
     )
-    def from_tuples(cls, data, closed="right", copy=False, dtype=None):
+    def from_tuples(
+        cls, data: Sequence[Tuple[Any, Any]], closed="right", copy=False, dtype=None
```

Comment:
is Tuple[Any, Any] accurate here? isnt it something like (Timestamp, Timestmap), (Timedelta, Timedelta), or (Numeric, Numeric)?

annotate closed copy, and return type while we're here?

Reply:
not yet. will close and pull these changes at a later date after other hints are added.

the `Tuple[Any, Any]` was just to ensure a two-tuple at this stage.